Utilities/gdcmcharls/CMakeLists.txt

```diff
@@ -1,5 +1,3 @@
-cmake_minimum_required(VERSION 2.8.7)
```

Comment:
I was removing them because this cmake call also changes the default policies that are enforced, and that greatly complicates building (in the test case with LTO optimizations).  Everything that follows the cmake_minimum_required(VERSION 2.8.7) will use only those policies.  

We could either have those lines commented out (as a place holder for the diff updates), or add:
```
set(GDCM_MAX_VALIDATED_CMAKE_VERSION "3.13.1")
if("${CMAKE_VERSION}" VERSION_LESS_EQUAL "${GDCM_MAX_VALIDATED_CMAKE_VERSION}")
  # As of 2018-12-04 GDCM has been validated to build with cmake version 3.13.1 new policies.
  # Set and use the newest cmake policies that are validated to work
  cmake_policy(VERSION ${CMAKE_VERSION})
endif()
```

below each of those function calls.

My intended behavior was that when GDCM cmake enviroinment is configured, it should be consistent throughout the enire code base.